jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/component/AbstractGridLoader.java

```diff
@@ -143,10 +126,14 @@ protected void loadData() {
                 fetchPlan = getFetchPlanRepository().getFetchPlan(holder.getMetaClass(), FetchPlan.LOCAL);
             }
 
-            loadColumns(resultComponent, columns, holder, fetchPlan);
+            loadColumns(resultComponent, columns, holder.getMetaClass(), fetchPlan);
```

Comment:
Previously, a holder was needed, because their data loader was required.

Now the data loader is not needed to load the filterable column, so passing the holder is redundant.